shared/constants/types/wallets.js

```diff
@@ -17,7 +17,7 @@ export opaque type AccountID: string = string
 export const stringToAccountID = __DEV__
   ? (s: string): AccountID => {
       if (!s) {
-        throw new Error('Invalid empty converationidkey. Did you mean Constants.noConversationIDKey?')
+        throw new Error('Invalid empty AccountID. Did you mean Constants.noAccountID?')
```

Comment:
-> I hope you didn't mean Constants.noAccountID cause there's no such thing

Reply:
hm I just meant this thing: https://github.com/keybase/client/blob/b91a6d5fd20758c5a3428cc8d2a012eab4b13e7a/shared/constants/types/wallets.js#L29

I'm not sure how these symbols are exported

Reply:
Oh there it is. My bad!